### ISSUE TITLE
Add historical events for when things were legalized/decriminalized

### ISSUE THREAD

**Issue by VickiLanger** [VickiLanger/LGBTQ-of-the-day-bot#54]
I would love to see an event for each US state like the below:

`'2009-04-03': 'Same-sex marriage is legal in the state of Iowa. https://en.wikipedia.org/wiki/Same-sex_marriage_in_Iowa',`

In addition, it would be great to have events like that for all the countries that have made things legal.

---

All events added should look like the below
```python
    '2020-10-04': 'title',
    '1969-06-28': 'Stonewall riots https://en.wikipedia.org/wiki/Stonewall_riots',
    '2019-09-16', 'words that will be tweeted and a reference link url',
```

### Proposed workflow:
1. Fork this repo
2. Add events to `words_dir/historical_events.py` keeping to the existing style/conventions:
   * Each event should have a date, some text, and a reference link
   * follow the examples above
3. Once you are done and checked for typos, commit it & push your work to your fork
4. Submit a PR against this repo

Someone will look it over and, if deemed appropriate, will merge your PR.

Thanks for contributing!

---
*Don't forget to follow the bot on [Twitter](https://www.twitter.com/LGBTQotd) to see the hard work you put in.*

**Comment by yogmel** [comment]
Hi @VickiLanger !
I would like to contribute if possible :)

**Comment by VickiLanger** [comment]
@yogmel You're welcome to contribute

**Comment by midhun1998** [comment]
Hi @VickiLanger. I would also like to contribute. Can I add an event for country India?

**Comment by VickiLanger** [comment]
@midhun1998 Yes, absolutely. Please do.

**Comment by dylan-nogueras** [comment]
Hi, @VickiLanger! I would like to add some events from Spain. Is that okay?

**Comment by VickiLanger** [comment]
Yes, please do. I would love to see events from all over the world. Have fun, @aure-nogueras
